draft-ietf-quic-qlog-h3-events.md

```diff
@@ -388,8 +403,10 @@ HTTPFrame =  HTTPDataFrame /
              HTTPMaxPushIDFrame /
              HTTPReservedFrame /
              HTTPUnknownFrame
+
+$HTTPFrame /= HTTPBaseFrames
 ~~~
-{: #httpframe-def title="HTTPFrame definition"}
+{: #httpbaseframe-def title="HTTPFrames defined in this document"}
```

Comment:
I prefer the old title, its already clear the things in this document are defined in this document.